tests/test_spend.py

```diff
@@ -9,6 +9,15 @@
 )
 
 
+def additional_fees(anc_vsize, anc_fee, target_feerate):
+    """The additional fee which must have been computed by lianad."""
+    computed_anc_vsize = int(anc_fee / target_feerate)
+    print(f"c  ", computed_anc_vsize)
```

Comment:
trailing debugging trace :)

EDIT: oh, it was in the original. We should still drop those prints IMO.

Reply:
Have dropped these now.